aiida/restapi/resources.py

```diff
@@ -273,6 +273,14 @@ def get(self, id=None, page=None):  # pylint: disable=redefined-builtin,invalid-
             headers = self.utils.build_headers(url=request.url, total_count=0)
             results = self.trans.get_namespace()
 
+        elif query_type == 'full_types_count':
+            headers = self.utils.build_headers(url=request.url, total_count=0)
```

Comment:
where is `headers` used?

Reply:
I honestly have no idea, I just copied this from the previous section for `'full_types'` (and also a bit from `'statistics'`, that is why I used the `usr` variable). It is returned at the end of this function so I guess you need to set them?